conan/api/output.py

```diff
@@ -285,10 +286,14 @@ def flush(self):
         self.stream.flush()
 
 
-def cli_out_write(data, fg=None, bg=None, endline="\n", indentation=0):
+def cli_out_write(data, fg=None, bg=None, endline="\n", indentation=0, filename=None):
     """
     Output to be used by formatters to dump information to stdout
     """
+    if filename is not None:
+        ConanOutput().info(f"Formatted output saved to '{filename}'")
+        save(filename, data)
```

Comment:
There are a few commands whose formatter calls `cli_out_write` multiple times in a row.

Should we convert those to only call it once, or should this new functionality support appending?

Reply:
Good catch. Not sure to be honest, I tend to think to re-write the formatter to include only 1 call, but there are some case that use colors and that would be a bit less elegant, but still doable